### PR TITLE
Changed the initial scan type load to a request/response [#643]

### PR DESCRIPTION
**Pull request by mcpierce**
## Status
READY

## Does this PR contain migrations?
NO

## Before You Submit Your PR:
- [X] Have you announced your PR on the comixed-dev mailing list?
- [X] New feature (non-breaking change which adds functionality)
- [X] My code follows the code style of this project.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

**Comment by mcpierce** [comment]
Please note that this PR requires comixed/comixed#664 be merged first.

File: comixed-web/src/app/library/services/comic-list.service.spec.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ import { Frame, Subscription } from 'webstomp-client';
 import {
   COMIC_LIST_UPDATE_TOPIC,
   LOAD_SCAN_TYPES_MESSAGE,
-  SCAN_TYPE_ADD_QUEUE
+  SCAN_TYPE_UPDATE_TOPIC
```
Comment (BRUCELLA2) [inline comixed-web/src/app/library/services/comic-list.service.spec.ts]: This imports are not used here

Comment (mcpierce) [reply comixed-web/src/app/library/services/comic-list.service.spec.ts]: It's being used in ScanTypeService as part of the subscription to scan type updates:

```
        this.logger.trace('Subscribing to scan type updates');
        this.subscription = this.webSocketService.subscribe<ScanType>(
          SCAN_TYPE_UPDATE_TOPIC,
          scanType => {
            this.logger.debug('Received scan type:', scanType);
            this.store.dispatch(scanTypeAdded({ scanType }));
          }
        );
        this.webSocketService.send(LOAD_SCAN_TYPES_MESSAGE, '');
      }
```

Comment (mcpierce) [reply comixed-web/src/app/library/services/comic-list.service.spec.ts]: Removed the unneeded imports.

**Review by BRUCELLA2 — APPROVED** [review]

**Pull request merged**